### ISSUE TITLE
Virtual buoy - drop by lat/lon

### ISSUE THREAD

**Issue by kbailey-noaa** [asascience-open/IOOS-EDS#164]
From the OFS modeler demo - someone asked about the option to drop a virtual buoy by entering lat/lon instead of clicking on the map.  Would like to add this feature.

**Comment by kbailey-noaa** [comment]
Potentially part of 2.0.3 because I thought I heard RPS is already working on this.  Consider it lower priority - I'm open to moving it out of that release.

**Comment by Bobfrat** [comment]
@kbailey-noaa on http://dev.oceansmap.com/eds/ if you create a Virtual Station, you'll see a link in the table next the the title to Update coordinates. Let me know if this suits the need.

<img width="451" alt="screen shot 2018-10-31 at 10 51 22 am" src="https://user-images.githubusercontent.com/5702672/47796551-f24f0600-dcfa-11e8-9319-01b330a4fde1.png">


<img width="678" alt="screen shot 2018-10-31 at 10 51 32 am" src="https://user-images.githubusercontent.com/5702672/47796558-f4b16000-dcfa-11e8-9a2d-15edabf12a88.png">

**Comment by kbailey-noaa** [comment]
Functionally it works well and suits the need.  It's a bit hidden, though, and you have to drop a virtual buoy first in order to change the location.  I think the users who were asking for it (that I heard in our NOS modeling meeting a couple months ago) were looking for something more exposed where you can just type on lat/long and hit enter and it drops a virtual buoy.   If you'd like I can solicit some folks to test it and see if this meets their need though.  i.e. we can implement now and tweak later.  It meets the functional requirement, regardless.

**Comment by Bobfrat** [comment]
@kbailey-noaa the more feedback the better! For other implementations the question would be, how does a user invoke the creation of the Virtual Station? Is there a button on the TOC? In a menu? That was a challenge we were facing and ultimately decided that not changing the current workflow was the path of least resistance yet still gets the functionality in.

**Comment by kbailey-noaa** [comment]
@Bobfrat @kknee Actually this is great timing since CO-OPS testers will be using the IOOS Model Viewer for LMHOFS testing.  As part of that announcement I can point that feature out to get feedback on that functionality.  

I just looked at the dev version realized that "Update coordinates" makes it sound like the original coordinates were wrong.  I think "New coordinates" would represent better what we're offering the users.  i.e. it makes it more apparent they have the option to change coords.  If you agree can you change that?  And then please push to production.

**Comment by Bobfrat** [comment]
Text has been updated on dev to ready "New coordinates"

**Comment by Bobfrat** [comment]
Text has been updated on production to ready "New coordinates"

**Comment by kbailey-noaa** [comment]
Great!  Will close this, and once LMHOFS is ready I'll ask users to take a look at this feature as well.
